### PR TITLE
PH tidy up

### PR DESCRIPTION
**Pull request by FerretDave**
cleaned up Monk abilities and improved some descriptions/corrected some typos Fixed dwarven war axe to be single handed (for a dwarf to use) Expanded 'USABLE_ITEM' option for some larger items.

### CODE REVIEW

File: data/35e/wizards_of_the_coast/core/players_handbook/ph_equip.lst [hunk 1]
```diff
@@ -131,7 +131,7 @@ Sword, Short			KEY:Sword (Short)				PROFICIENCY:WEAPON|Sword (Short)			TYPE:Weap
...
-Waraxe, Dwarven			KEY:Waraxe (Dwarven)			PROFICIENCY:WEAPON|Waraxe (Dwarven)			TYPE:Weapon.Container.Melee.Martial.Exotic.Standard.Dwarven.Slashing.Axe												CONTAINS:UNLIM|Weapon Crystal=1											COST:30	WT:8							CRITMULT:x3	CRITRANGE:1							DAMAGE:1d10				EQMOD:STEEL						WIELD:TwoHanded												SIZE:M	SOURCEPAGE:p.121																							BONUS:WEAPON|WIELDCATEGORY|-1|PREVARGTEQ:DwarvenWaraxeExoticUse,1																		SPROP:A dwarf treats a dwarven waraxe as a martial weapon even when using it in one hand.
+Waraxe, Dwarven			KEY:Waraxe (Dwarven)			PROFICIENCY:WEAPON|Waraxe (Dwarven)			TYPE:Weapon.Container.Melee.Martial.Exotic.Standard.Dwarven.Slashing.Axe												CONTAINS:UNLIM|Weapon Crystal=1											COST:30	WT:8							CRITMULT:x3	CRITRANGE:1							DAMAGE:1d10				EQMOD:STEEL						WIELD:OneHanded	SIZE:M	SOURCEPAGE:p.121																							BONUS:WEAPON|WIELDCATEGORY|-1|PREVARGTEQ:DwarvenWaraxeExoticUse,1																		SPROP:A dwarf treats a dwarven waraxe as a martial weapon even when using it in one hand.
```
Comment (BahamutDragon) [inline data/35e/wizards_of_the_coast/core/players_handbook/ph_equip.lst:134]: Why are we changing this weapon?

**Review by BahamutDragon — CHANGES_REQUESTED** [review]
Dwarven Waraxe change does not match description of item.

File: data/35e/wizards_of_the_coast/core/players_handbook/ph_equip.lst (hunk 1 above)
Comment (FerretDave) [reply data/35e/wizards_of_the_coast/core/players_handbook/ph_equip.lst:134]: a dwarf can use it one handed, as per the SPROP, so this allows it to be equipped appropriately for a dwarf (a non-dwarf is unlikely to equip it really, though possibly a PRERACE could work?)

Comment (BahamutDragon) [reply data/35e/wizards_of_the_coast/core/players_handbook/ph_equip.lst:134]: The weapon already shifts from TwoHanded to OneHanded if the character is a dwarf. Much like the Bastard Sword shifts to Onehanded if the character has Exotic Weapon Bastard Sword feat. Unless the mechanic is not working, we should not just change one aspect of the item. As written, the weapon will become a LIGHT weapon for a Dwarf. 

That is why this tag exists: BONUS:WEAPON|WIELDCATEGORY|-1|PREVARGTEQ:DwarvenWaraxeExoticUse,1

Comment (FerretDave) [reply data/35e/wizards_of_the_coast/core/players_handbook/ph_equip.lst:134]: now that's perplexing, so it works now (just reverted the change and retested), previously I'd been unable to equip it two handed for a dwarf (to get the 1.5 strength mod) without this change.
Apologies for this, revering this one now.

**Pull request merged**